### ISSUE TITLE
Getting Error - No such file or directory @ rb_sysopen 

### ISSUE THREAD

**Issue by UnmeshDeshmukh** [pmq20/node-packer#137]
/__enclose_io_memfs__/lib/ruby/2.4.0/fileutils.rb:1289:in `initialize': No such file or directory @ rb_sysopen - path.exe (Errno::ENOENT)

Using the command nodec src/ --skip-npm-install --quiet -o temp/ --tmpdir= ../etmp --clean-tmpdir --msi

Running this command from a linux debian package. I want to create a windows executable for the node project please let me know if that issue is fixed or not

**Comment by slee047** [comment]
Did you install gcc, g++, make, ruby?

**Comment by UnmeshDeshmukh** [comment]
Yes I have installed all. I am able to generate a.out file but when I specify the option --msi it throws an error

**Comment by UnmeshDeshmukh** [comment]
> Did you install gcc, g++, make, ruby?

Yes I have installed all. I am able to generate a.out file but when I specify the option --msi it throws an error
